backend-worker/worker.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import logging
+import time
+
+import requests
+
+logging.basicConfig(level=logging.INFO)
+LOGGER = logging.getLogger(__name__)
+
+
+def do_work(ip, port):
+    LOGGER.info(f'Starting worker, queueing {ip}:{port}')
+    while True:
+        #response = requests.get(f'{ip}/jobsqueue/:{port}')
```

Comment:
Just an FYI that I noticed with testing locally. This will need to be `f'{ip}:{port}/jobsqueue`.